<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRepoDlg</class>
 <widget class="QDialog" name="CreateRepoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new repository</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="m_DisableFsync">
       <property name="text">
        <string>Disable fsync at commit (BDB only)</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Path to repository:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="m_FilesystemSelector">
       <property name="toolTip">
        <string>Select type of storage</string>
       </property>
       <property name="whatsThis">
        <string>Select the storage type of repository (FSFS or Berkely DB)</string>
       </property>
       <item>
        <property name="text">
         <string>FSFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BDB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="m_CreateMainDirs">
       <property name="toolTip">
        <string>Create trunk, tags and branches folders</string>
       </property>
       <property name="whatsThis">
        <string>If this is set then the base layout (&lt;tt&gt;/trunk&lt;/tt&gt;,&lt;tt&gt;/branches&lt;/tt&gt; and &lt;tt&gt;/tags&lt;/tt&gt;) will created after opening the fresh repository.</string>
       </property>
       <property name="text">
        <string>Create main folders</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="m_presvn16compat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Is created repository compatible to Subversion prior 1.6</string>
       </property>
       <property name="whatsThis">
        <string>If set, the repository created will compatible to Subversion prior 1.5. This is only useful when svnqt is running with Subversion 1.6 or above.</string>
       </property>
       <property name="text">
        <string>Compatible to Subversion prior 1.6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="m_LogKeep">
       <property name="text">
        <string>Disable automatic log file removal (BDB only)</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Type of repository:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QCheckBox" name="m_presvn15compat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Is created repository compatible to Subversion prior 1.5</string>
       </property>
       <property name="whatsThis">
        <string>If set, the repository created will compatible to Subversion prior 1.5. This is only useful when svnqt is running with Subversion 1.5 or above.</string>
       </property>
       <property name="text">
        <string>Compatible to Subversion prior 1.5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="KUrlRequester" name="m_ReposPathinput">
       <property name="mode">
        <set>KFile::Directory|KFile::LocalOnly</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="m_presvn18compat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Is created repository compatible to Subversion prior 1.8</string>
       </property>
       <property name="whatsThis">
        <string>If set, the repository created will compatible to Subversion prior 1.4. This is only useful when svnqt is running with Subversion 1.8 or above.</string>
       </property>
       <property name="text">
        <string>Compatible to Subversion prior 1.8</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_ReposPathinput</tabstop>
  <tabstop>m_FilesystemSelector</tabstop>
  <tabstop>m_DisableFsync</tabstop>
  <tabstop>m_LogKeep</tabstop>
  <tabstop>m_CreateMainDirs</tabstop>
  <tabstop>m_presvn15compat</tabstop>
  <tabstop>m_presvn16compat</tabstop>
  <tabstop>m_presvn18compat</tabstop>
 </tabstops>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
 <slots>
  <slot>compatChanged15(bool)</slot>
 </slots>
</ui>
